src/assets/scripts/client/airline/buildFlightNumber.js

```diff
@@ -0,0 +1,85 @@
+import { choose } from '../utilities/generalUtilities';
+import { DEFAULT_CALLSIGN_FORMAT } from '../constants/airlineConstants';
+
+const ALPHA = 'abcdefghijklmnopqrstuvwxyz';
+const NUMERIC = '0123456789';
+const defaultCallsignFormats = [DEFAULT_CALLSIGN_FORMAT];
+
+/**
+ * This picks a random number. If it is the first value within the callsign (ie. i === 0), then it picks a
+ * number between 1 and 9. Otherwise, it picks a number between 0 and 9.
+ * @function _generateRandomDigit
+ * @param i {number}
+ * @return NUMERIC {string}
+*/
+function _generateRandomDigit(i) {
+    if (i === 0) {
+        return choose(NUMERIC.substr(1));
+    }
+
+    return choose(NUMERIC);
+}
+
+/**
+ * This function picks a random letter and returns it.
+ * @function _generateRandomLetter
+ * @return ALPHA {string}
+*/
+function _generateRandomLetter() {
+    return choose(ALPHA);
+}
+
+/**
+ * This function takes a list of callsignFormats, validates them (ie. ensures that no callsignFormat has a leading 0)
+ * then returns a list of validated callsignFormats or the default callsignFormat if there are no valid callsignFormats
+ * @function _validateCallsignFormats
+ * @param callsignFormat {Array}
```

Comment:
`Array` should be lowercase.  

If you can, also add the type expected in the array: `{array<string>}` or `{array<string | number>}`